common/test/acceptance/tests/lms/test_lms.py

```diff
@@ -1012,7 +1012,7 @@ def test_python_execution_in_problem(self):
         self.course_nav.go_to_section('Test Section', 'Test Subsection')
 
         problem_page = ProblemPage(self.browser)
-        self.assertEqual(problem_page.problem_name, 'PYTHON PROBLEM')
+        self.assertEqual(problem_page.problem_name, 'Python Problem')
```

Comment:
I would change the thing that's being checked instead of the thing being checked against (unless we care about testing for a specific casing).

(e.g. `self.assertEqual(problem_page.problem_name.upper() , 'PYTHON PROBLEM')` or `self.assertEqual(problem_page.problem_name.lower() , 'python problem)`)

That would prevent having to update these tests in the future if we change the case of the text.